.eslintrc.json

```diff
@@ -1,4 +1,5 @@
 {
+  "ignorePatterns": ["types/", "dist/", "lib/"],
```

Comment:
I see this is useful when running eslint directly, e.g. `npm exec -- eslint .` and not `npm run lint`

Reply:
Also useful when running an eslint plugin within an editor such as Visual Studio Code.